.eslintrc.js

```diff
@@ -328,6 +332,58 @@ module.exports = {
 
     'unused-imports/no-unused-imports': 'error',
     'unused-imports/no-unused-vars': 'off',
+    // New rules added to unicorn
+    'unicorn/no-array-reduce': 'off',
+    'unicorn/no-array-callback-reference': 'off',
+    'unicorn/numeric-separators-style': 'off',
+    'unicorn/prefer-switch': 'off',
+    'unicorn/prefer-spread': 'off',
+    'unicorn/template-indent': 'off',
+    'unicorn/prefer-export-from': 'off',
+    'unicorn/explicit-length-check': 'off',
+    'unicorn/prefer-date-now': 'off',
+    'unicorn/no-await-expression-member': 'off',
+    'unicorn/prefer-dom-node-text-content': 'off',
+    'unicorn/consistent-destructuring': 'off',
+    'unicorn/no-new-array': 'off',
+    'unicorn/prefer-ternary': 'off',
+    'unicorn/prefer-dom-node-dataset': 'off',
+    'unicorn/no-useless-spread': 'off',
+    'unicorn/no-lonely-if': 'off',
+    'unicorn/prefer-array-some': 'off',
+    'unicorn/prefer-native-coercion-functions': 'off',
+    'unicorn/relative-url-style': 'off',
+    'unicorn/prefer-array-flat-map': 'off',
+    'unicorn/prefer-node-protocol': 'off',
+    'unicorn/prefer-module': 'off',
+    'unicorn/no-useless-promise-resolve-reject': 'off',
+    'unicorn/no-array-method-this-argument': 'off',
+    'unicorn/no-array-for-each': 'off',
+    'no-unsafe-optional-chaining': 'off',
+    'unicorn/import-style': 'off',
+    'unicorn/no-array-push-push': 'off',
+    'unicorn/no-useless-switch-case': 'off',
+    'unicorn/prefer-regexp-test': 'off',
+    'unicorn/no-object-as-default-parameter': 'off',
+    'unicorn/prefer-code-point': 'off',
+    'unicorn/prefer-object-from-entries': 'off',
+    'unicorn/prefer-math-trunc': 'off',
+    'unicorn/text-encoding-identifier-case': 'off',
+    'unicorn/no-unreadable-iife': 'off',
+    'unicorn/error-message': 'off',
+    'unicorn/no-thenable': 'off',
+    'unicorn/prefer-array-index-of': 'off',
+
+    '@typescript-eslint/no-unsafe-argument': 'off',
+    '@typescript-eslint/no-unnecessary-type-constraint': 'off',
+
+    // End
+
+    // Previous Rules that has been causing errors after upgrading
+    'import/extensions': 'off',
+    'unicorn/consistent-function-scoping': 'off',
+    'rxjs/no-nested-subscribe': 'off',
```

Comment:
Could you share information on errors you're getting for each rule as a comment here?

Reply:
Added on comment [above](https://github.com/sourcegraph/eslint-config/pull/247#discussion_r877756140)